src/flinkSql/languageClient.ts

```diff
@@ -92,20 +99,38 @@ export async function initializeLanguageClient(
             },
           },
           initializationFailedHandler: (error) => {
-            logger.error(`Language server initialization failed: ${error}`);
+            let msg = "Language client initialization failed";
+            logger.error(msg, JSON.stringify(error));
+            logError(error, msg, {
+              extra: {
+                wsUrl: url,
+              },
+            });
             return true; // Don't send the user an error, we are handling it
           },
           errorHandler: {
             error: (error: Error, message: Message): ErrorHandlerResult => {
-              logger.error(`Language server error: ${message}`);
+              let msg = "Language client error handler invoked.";
+              logger.error(msg, JSON.stringify(error));
+              logError(error, msg, {
+                extra: {
+                  wsUrl: url,
+                },
+              });
               return {
                 action: ErrorAction.Continue,
                 message: `${message ?? error.message}`,
                 handled: true, // Don't send the user an error, we are handling it
               };
             },
             closed: () => {
-              logger.warn("Language server connection closed by the client's error handler");
+              let msg = "Language client connection closed by the client's error handler";
+              logger.warn(msg);
+              logError(new Error(msg), msg, {
+                extra: {
+                  wsUrl: url,
+                },
+              });
```

Comment:
nit: Assuming this code is called whenever the WebSocket session is closed by the remote server, should we log it as usage (if at all) not error? Otherwise, this might create a lot of noise, as we expect a lot of closures due to the aggressive idle timeout.